ush/wave_outp_cat.sh

```diff
@@ -52,12 +51,14 @@
 
   echo "   Generate input file for ww3_outp."
 
-  if [ "$specdir" = "bull" ]
+  if [[ "$specdir" == "bull" ]]
   then
     outfile=${STA_DIR}/${specdir}/$WAV_MOD_TAG.$buoy.bull
     coutfile=${STA_DIR}/c${specdir}/$WAV_MOD_TAG.$buoy.cbull
     for f in outfile coutfile; do
-      if [[ -f ${f} ]]; then rm ${f}; fi
+      if [[ -f "${f}" ]]; then
+         rm "${f}"
```

Comment:
```suggestion
         rm -f "${f}"
```

Reply:
Fixed; this was actually a hidden bug.  It should have been `rm "${!f}".  As it was, it was removing 'outfile' and 'coutfile', which never existed instead of ${coutfile} and ${outfile}.